packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```
@@ -0,0 +1,25 @@
+{
+  "pci_database_add_title": "Créer une base de données",
+  "pci_database_add_description": "Créez une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, veuillez <a href='{{createPrivateNetworkLink}}'>créer un réseau privé</a> ou un sous-réseau d’abord.",
```

In frensh language we should avoid to use imperative word (ez) but instead:
```suggestion
  "pci_database_add_description": "Créer une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, veuillez <a href='{{createPrivateNetworkLink}}'>créer un réseau privé</a> ou un sous-réseau d’abord.",
```

---

Corrected. thanks!